### ISSUE TITLE
please upgrade to device_info_plus 8.0.0

### ISSUE THREAD

**Issue by renanyoy** [ScerIO/packages.flutter#365]
pdfx still using old  device_info_plus 4.0.0

**Comment by 2shrestha22** [comment]
It still uses 4.0.0.

**Comment by renanyoy** [comment]
I replaced by pdf render cause of that, and had 20 addons which hava upgraded directly, they were blocked  since months with pdfx..

**Comment by hedbladucf** [comment]
Is anyone maintaining this project?

**Comment by Gvozdik-Darya** [comment]
still using old device_info_plus

**Comment by wibtu** [comment]
My pull request for the update of the device_info_plus package to version 8.0.0 was accepted. So hopefully we will get an updated version soon.

**Comment by hschk** [comment]
Any news? Is there an ETA for this update?

**Comment by michaelsoliman1** [comment]
Can any of the maintainers please update the pub package to the latest version fixing this issue ? cc/ @SergeShkurko 

I had to download the whole repo to use the latest changes

**Comment by mreinfurt** [comment]
Yeah, it's already been changed in https://github.com/ScerIO/packages.flutter/pull/377 so a new release would be much appreciated. 👍

**Comment by sgehrman** [comment]
this project is difficult to fork.  How can I fork 2.3.0, and update device_info_plus?  I have no idea if the main branch is safe to use.  clean up the code and use branches or something.

**Comment by AncientPixel** [comment]
Is this still going to happen... it's been months

**Comment by wibtu** [comment]
Because there is no visible progress and Serge seems to be dissapeared and not responding to mails or messages.
Is there anybody willing to fork this repository and publish a new version on pub.dev?

**Comment by austinstoker** [comment]
I'm not willing to fork it, but I got things working for me by adding this to pubspec.yaml
<pre>
dependency_overrides:
  device_info_plus:
    version: ^8.0.0
</pre>
